executor/tiflash_test.go

```diff
@@ -139,3 +139,56 @@ func (s *tiflashTestSuite) TestMppExecution(c *C) {
 	// test proj and selection
 	tk.MustQuery("select count(*) from (select a * 2 as a from t1) t1 , (select b + 4 as a from t)t where t1.a = t.a").Check(testkit.Rows("3"))
 }
+
+func (s *tiflashTestSuite) TestPartitionTable(c *C) {
```

Comment:
1. suggest to do a mannul test on tpch with different table partitions.
2. add some null data for integration tests
3. test some empty table for integration tests
4. test partitioned table works with non-partitioned tables.

Reply:
1. The integration test for partition table which is based on TPC-H will be added after merging. For now, the manual test can work well.
2. good suggestion.
3. empty table is meaningless, but empty partition is nessesary, which have been included in the tests.
4. good suggestion.